### ISSUE TITLE
Game crashes on maximising window or loading a save

### ISSUE THREAD

**Issue by Mister-Cultist** [CleverRaven/Cataclysm-DDA#32406]
After updating my game today (not sure if it was an update earlier because I was working today) my game would start in a small window instead of the usual bigger one, after maximising it the game would crash. I tried loading a game while leaving the screen windowed but it crashed again. I deleted all my game files and redownloaded the launcher and game, the same problem happened again. Any attempt to resize the screen gives me this message:
" Failed to write options to "./config/options.json": moving temporary file ".config/options.json.10680.temp" failed "

**Comment by RogueYun** [comment]
I can't even create a world now... it gives me 'Failed to write world data to "./save/WorldName/worldoptions.json": moving temporary file "./save/WorldName/worldoptions.json. 12992.temp" failed'

**Comment by AMurkin** [comment]
Duplicate of #32401
#32404 should fix issue with saving.
`game would start in a small window` - options.json already renamed/corrupted at this point. You may rename it in the config folder after the fix.

**Comment by b0urgeoisie** [comment]
still experiencing this issue after #32404 

any sort of write or temp file move causes the error

**Comment by ZhilkinSerg** [comment]
Duplicate of #32401
